src/zope/app/apidoc/static.py

```diff
@@ -449,6 +474,27 @@ def _create_arg_parser():
     parser.add_argument("target_dir",
                         help="The directory to contain the output files")
 
+    ######################################################################
+    # ZCML
+
+    zcml = parser.add_argument_group(title='ZCML',
+                                          description='Options that specify what ZCML file to load')
+    
+    zcml.add_argument(
```

Comment:
Since the webserver is already in a mutually exclusive group with the publisher backend, I don't think you can add it to a mutually exclusive group with `zcml_file`. I'm not sure how to declare argument dependency without using [subcommands](https://docs.python.org/3/library/argparse.html#sub-commands).

Reply:
*has a look at the existing args*

I could see this fitting fine into the existing `ret_kind` mutually exclusive group. Either you use the default publisher (-p, which shows the default Zope 3 configuration), or you use a custom publisher configuration (this new option), or you connect to an external server to spider.